### PR TITLE
Add back-channel logout endpoint and iat validation configs

### PR DESCRIPTION
**Pull request by ThaminduR**
### Proposed changes in this pull request

Add back-channel logout endpoint.
Add `iat` validation configurations.

Related to - https://github.com/wso2/product-is/issues/9254

### CODE REVIEW

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/repository/conf/identity/identity.xml
Comment (piraveena) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/repository/conf/identity/identity.xml:951]: add the config to .j2 file as well

Comment (ThaminduR) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/resources/repository/conf/identity/identity.xml:951]: 7536f23

**Comment by piraveena** [comment]
need to merge this PR after oidc authenticator PR got merged

**Review by piraveena — APPROVED** [review]

**Pull request merged**
